### PR TITLE
Complete `Operator` of Python API

### PR DESCRIPTION
**Pull request by Canpio**

### CODE REVIEW

File: python/paddle/v2/framework/graph.py [hunk 1]
```diff
@@ -106,30 +107,119 @@ def _convert_np_dtype_to_dtype_(np_dtype):
...
-                 inputs=None,
-                 outputs=None,
+    def __init__(self, block, desc, type, inputs=None, outputs=None,
                  attrs=None):
         self.block = block
         self.desc = desc
-        if type is not None:
-            # TODO.
-            pass
+        self.proto = OpProtoHolder.instance().get_op_proto(type)
```
Comment (reyoung) [inline python/paddle/v2/framework/graph.py]: I think an operator might not have operator proto, such as operators generated during backward.

Comment (Canpio) [reply python/paddle/v2/framework/graph.py]: Yes, but backward operators will not be created by Python API.

File: paddle/framework/op_desc.cc [hunk 2]
```diff
@@ -100,6 +100,12 @@ void OpDescBind::SetAttr(const std::string &name, const Attribute &v) {
   need_update_ = true;
 }
 
+void OpDescBind::SetBlockAttr(const std::string &name, BlockDescBind &block) {
+  BlockDesc *desc = block.RawPtr();
+  this->attrs_[name] = desc;
```
Comment (jacquesqiao) [inline paddle/framework/op_desc.cc]: OpDesc为啥要把BlockDesc放进去？

Comment (Canpio) [reply paddle/framework/op_desc.cc]: Rnn Op and IfElse Op can hold their own Block.

Comment (jacquesqiao) [reply paddle/framework/op_desc.cc]: got it.

**Pull request merged**